.github/actions/detect-workflow-js/src/detect.ts

```diff
@@ -62,11 +62,12 @@ export async function detectWorkflowFromContext(
 
   let [repository, ref, workflow] = ["", "", ""];
 
-  // If this is a slsa-github-generator repository or fork, then look
-  // for the repo and head SHA from the pull_request event value.
+  // If this is a pull request, then look for the repo and head SHA from the
```

Comment:
nit: add back the comment about slsa-github-generator, since the condition checks for it.

Reply:
Done.